package.json

```diff
@@ -40,7 +40,7 @@
     "standalone:replay": "REPLAY=replay cozy-konnector-standalone src/index.js",
     "build": "webpack",
     "format": "prettier --write '**/*.{js,json}'",
-    "precommit": "eslint . && pretty-quick --staged",
```

Comment:
Then this should be generalized in other connectors

Reply:
Yes I've done that